features/notifications/NotificationsModels.ts

```diff
@@ -0,0 +1,9 @@
+type Notification = {
+  id: string;
+  userId: string;
+  message: string;
+  createdAt: string;
+  seen: boolean;
+};
+
+export type { Notification as TNotification };
```

Comment:
Why did you name this TNotification?

Reply:
Because Notification is predefined and I couldn't use it.